apps/site/lib/site_web/views/layout_view.ex

```diff
@@ -41,11 +41,20 @@ defmodule SiteWeb.LayoutView do
   def nav_link_content(conn),
     do: [
       {"Getting Around", "Transit Services, Plan Your Journey, Riding...",
-       static_page_path(conn, :getting_around)},
+      static_page_path(conn, :getting_around)},
       {"Fares", "Fares By Mode, Reduced Fares, Passes...", cms_static_page_path(conn, "/fares")},
       {"Contact Us", "Phone And Online Support, T-Alerts", customer_support_path(conn, :index)},
       {"More", "About Us, Business Center, Projects...", static_page_path(conn, :about)}
     ]
+    # When time to remove deskstop_nav_redesign feature flag replace the above with below
+    #   [
+    #     {"Transit", "Modes of Transit, Plan Your Journey, Find a Location...",
```

Comment:
I saw what you were referring to regarding the filename, oof that's kinda annoying.

One possibly not-so-elegant idea that should work: make files corresponding to any new names -- so for this section the template looks for a `_desktop_menu_transit.html.eex` file, so adding that and putting the new links there. Then doing the same for `_desktop_menu_contact.html.eex`, and `_desktop_menu_about.html.eex`.

Reply:
A more involved change might involve updating `_nav_desktop.html.eex` to

1. set the `filename` variable to something smarter than the `name` or
2. adjusting the rendering logic from `<%= render "_desktop_menu_#{filename}.html", conn: @conn, common_fare_summaries: fare_summaries %>` to something else.